manifests/init.pp

```diff
@@ -451,6 +452,20 @@
     }
   }
 
+  # Debian 9 use systemd
+  if ( $facts['os']['name'] == 'Debian' and versioncmp($facts['os']['release']['major'], '9') >= 0 ) {
+    file { 'snmpd.service':
```

Comment:
instead of using a file resource, can you please use [systemd::unit_file](https://github.com/camptocamp/puppet-systemd/#unit-files). Can you please place the file under `/etc/systemd/system/`? That is the place for custom service files.

Reply:
I can try, but I need help. This module needs a static file, but we need to make it dynamic sor I need a erb is this possible ? On debian 9 file is located in /lib/systemd/system/snmpd.service so I don't know what will happen if I deploy another file into `/etc/systemd/system`  ? Can we see that on slack  please ?

Reply:
hey. Sure, you can always ask in the voxpupuli channel on slack. You can use `epp()` to generate the content. FIles in `/etc/systemd/system/` overwrite those in `/lib/systemd/system/`.

Reply:
great will go this way then :)